detection-rules/attachment_any_html_new_sender.yml

```diff
@@ -10,7 +10,14 @@ type: "rule"
 severity: "medium"
 source: |
   type.inbound
-  and any(attachments, .file_extension in~ ('htm', 'html') or .file_type == "html")
+  and any(attachments,
+          (.file_extension in~ ('htm', 'html') or .file_type == "html")
+          
+          // negate cisco secure spawned HTML
+          and not regex.match(file.parse_html(.).raw,
```

Comment:
a little hesitant to negate on this, feels a little too easy to bypass

Reply:
thoughts on going back to the iphmx in headers.domains approach? or combined w/ message_id?

Reply:
Message ID and iphmx neither are not consistent in the group of FP's :(

Reply:
how many / what % don't match either of those patterns?